core/io/compression.cpp

```diff
@@ -33,6 +33,7 @@
 #include "core/config/project_settings.h"
 #include "core/io/zip_io.h"
 
+#include "thirdparty/lz4/lz4.h"
```

Comment:
```suggestion
#include <lz4.h>
```
To allow using both local and system version.